inventory/multi_inventory.py

```diff
@@ -160,13 +165,21 @@ def run_provider(self):
         '''
         try:
             all_results = []
-            tmp_file_paths = []
+            tmp_dir_paths = []
             processes = {}
             for account in self.config['accounts']:
-                env = account['env_vars']
-                if account.has_key('provider_config'):
-                    tmp_file_paths.append(MultiEc2.generate_config(account['provider_config']))
-                    env['EC2_INI_PATH'] = tmp_file_paths[-1]
+                tmp_dir = None
+                if account.has_key('provider_config_files'):
+                    tmp_dir = MultiInventory.generate_config(account['provider_config_files'])
```

Comment:
please rename to "provider_files" because it can be more than just config files.